pkg/analytics/test.go

```diff
@@ -42,6 +42,27 @@ type TestMetadata struct {
 	StagesCount int
 }
 
+// SingleTestMetadata contains the metadata of a single test execution
+type SingleTestMetadata struct {
+	// DevenvName is the name of the development environment where the test was run
+	DevenvName string `json:"devenv_name"`
```

Comment:
Same open question I left in the backend implementation. Does the dev env name provide some value in the metric? It might be misleading as we might have a name, but without having a dev environment deployed in the namespace

Reply:
Discussed offline about it. Dev env name is the only way we have to identify okteto manifests for the different operations and it is how we do it also for other operations as build, so it has to be specified